Tests/FTPropertyWrappersTests/KeychainCoderTest.swift

```diff
@@ -0,0 +1,151 @@
+import XCTest
+@testable import FTPropertyWrappers
+
+final class KeychainCoderTest: XCTestCase {
+
+    private class WrapperClass<T: Codable & Equatable>{
+
+        let wrapped: T
+        init(wrapped: T) {
+            self.wrapped = wrapped
+        }
+        func testCoding() throws {
+            let coder = KeychainCoding()
+            let data = try coder.encode(wrapped)
+            XCTAssertEqual(try coder.decode(T.self, from: data), wrapped)
+        }
+    }
+
+    private func assertCoding<T: Codable & Equatable>(input: T) throws {
+        try WrapperClass(wrapped: input).testCoding()
+    }
+
+    func testIntegers() {
+        do {
+            try assertCoding(input: Int(0))
+            try assertCoding(input: Int.max)
+            try assertCoding(input: Int.min)
+
+            try assertCoding(input: Int8(0))
+            try assertCoding(input: Int8.max)
+            try assertCoding(input: Int8.min)
+
+            try assertCoding(input: Int(0))
+            try assertCoding(input: Int16.max)
+            try assertCoding(input: Int16.min)
+
+            try assertCoding(input: Int32(0))
+            try assertCoding(input: Int32.max)
+            try assertCoding(input: Int32.min)
+
+            try assertCoding(input: Int64(0))
+            try assertCoding(input: Int64.max)
+            try assertCoding(input: Int64.min)
+        } catch {
+            XCTFail(error.localizedDescription)
+        }
+    }
+
+    func testUnsigned() {
+        do {
+            try assertCoding(input: UInt.max)
+            try assertCoding(input: UInt.min)
+
+            try assertCoding(input: UInt8.max)
+            try assertCoding(input: UInt8.min)
+
+            try assertCoding(input: UInt16.max)
+            try assertCoding(input: UInt16.min)
+
+            try assertCoding(input: UInt32.max)
+            try assertCoding(input: UInt32.min)
+
+            try assertCoding(input: UInt64.max)
+            try assertCoding(input: UInt64.min)
+        } catch {
+            XCTFail(error.localizedDescription)
```

Comment:
Let's asset that no coding should throw errors.

```suggestion
            XCTAssertNoThrow(throws error)
```